### PR TITLE
Add instructions on using libyaml

### PR DESCRIPTION
**Pull request by gwillem**
For a 10x speed increase!

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -573,6 +573,24 @@ If you set the loglevel to DEBUG, you will also get information about
 which matchers didn't match. This can help you with debugging custom
 matchers.
 
+Speed
+-----
+VCR.py runs about 10x faster when pyyaml can use the libyaml extensions. However, just installing ``libyaml`` (Mac) or ``libyaml-dev`` (Linux) is not enough, as pyyaml needs to be rebuild with the proper flag. Note that this flag is cached by pip, so clear the cache first. 
+
+Are you using libyaml already? This should work:
+
+::
```
Comment (IvanMalison) [inline README.rst]: mark these as sh/bash code?

**Comment by IvanMalison** [comment]
Looks good to me. Mind adding language annotations?

**Comment by agriffis** [comment]
Thanks!

**Comment by graingert** [comment]
Are these instructions up to date now that pyyaml provides wheels?

**Pull request merged**